Makefile

```diff
@@ -271,6 +271,7 @@ test-unit: guard-test
 	${DOCKER_COMPOSE_TEST} run --rm backend perl ${args} tests/unit/${test}
 
 # usage:  make test-int test=test-name.t
+# to update expected results: make test-int test="test-name.t --update-expected-results"
```

Comment:
I introduced args, so this may be better.

```suggestion
# to update expected results: make test-int test="test-name.t" args="--update-expected-results"
```

Reply:
en fait le args ça marche pas pour ça, ça passe un argument à perl (du genre -d), pas au test.
j'avais rajouté un params="--update-expected-results" en plus, et quand j'ai voulu updater la doc, j'ai vu qu'il y avait la méthode du "test-name.t --update-expected-results" qui était déjà documentée